src/data/index.ts

```diff
@@ -0,0 +1,25 @@
+import { getCurrentSemester } from './api';
+
```

Comment:
Is `index.ts` a good name for this file?

Reply:
Intended for being able to import `src/data`. By default, if you import a folder, it grabs the `index.js` file (or ts in this case). Also as was mentioned, I should rename this folder.